src/main/java/graphql/servlet/GraphQLExecutionResultHandlerFactory.java

```diff
@@ -0,0 +1,16 @@
+package graphql.servlet;
+
+import java.io.Writer;
+
+/**
+ * Interface to allow customization of how batched queries are handled.
+ */
+public interface GraphQLExecutionResultHandlerFactory {
```

Comment:
I am not sure about the name 'ExecutionResultHandler', is it handling ExecutionResults?